site/content/blogs/release-v131.en.md

```diff
@@ -18,9 +18,9 @@ Thanks to every one of you who shows love and support for AWS Copilot.
 
 Copilot v1.31 brings big enhancements to help you develop more flexibly and efficiently:
 
-- **NLB enhancements**: You can now add security groups to Copilot-managed NLBs. NLBs also support the UDP protocol.
+- **NLB enhancements**: You can now add security groups to Copilot-managed [network load balancers](../docs/manifest/lb-web-service.en.md#nlb). NLBs also support the UDP protocol.
 - **Better task failure logs**: Copilot will show more descriptive information during deployments when tasks fail, allowing better troubleshooting.
-- **`copilot deploy` enhancements: You can now deploy multiple workloads at once, or deploy all local workloads with `--all`.
+- **`copilot deploy` enhancements**: You can now deploy multiple workloads at once, or deploy all local workloads with `--all`.
 
```

Comment:
@iamhopaul123 do you want to include "import cert for static site"?

Reply:
Sure that's a good idea